machines/the-basics/getting-started.html.md.erb

```diff
@@ -0,0 +1,284 @@
+---
+title: Machines
+objective: Working with machines
+layout: framework_docs
+order: 3
+---
+
+Fly Machines are [Firecracker VMs](https://firecracker-microvm.github.io/) with a fast REST API that can boot instances in about 300ms, in any region supported by [Fly.io](https://fly.io/docs/../reference/regions/).
+
+The Machines API gives you efficient, low-level control over VM provisioning, supported by Fly.io infrastructure and networking features.
+
+Machines are also the spawning ground for new platform features like *wake-on-request* (also known as *scale-to-zero*). You can
+stop a running machine to save on compute costs. It then may be started automatically when a request arrives at the Fly proxy.
+
+This document covers usage of the Machines REST API to start, stop, update and interact with machines. For the impatient, [flyctl also provides commands](https://fly.io/docs/flyctl/machine/) for experimenting with the API.
+
+See all possible machine states [in the table below](#machine-states).
+
+## Connecting to the API
+
+The Machines API endpoint requires a connection to your Fly private network - either by running on a VM inside the network, or via a WireGuard VPN, or using `flyctl proxy`.
+
+This guide assumes that you have `flyctl` and `curl` installed, and have authenticated to the Fly.io platform.
+
+### Connecting via WireGuard
+
+This method requires more setup but gives you direct access to machines over the Wireguard VPN.
+
+First, follow the [instructions](/docs/../reference/private-networking/#private-network-vpn) to set up a permanent WireGuard connection to your Fly network. We recommend WireGuard because you can directly test your machines from your local machine.
```

Comment:
```suggestion
First, follow the [instructions](/docs/reference/private-networking/#private-network-vpn) to set up a permanent WireGuard connection to your Fly network. We recommend WireGuard because you can directly test your machines from your local machine.
```